lib/bb/tests/fetch.py

```
@@ -2753,8 +2814,8 @@ def test_npm_registry_alternate(self):
     @skipIfNoNpm()
     @skipIfNoNetwork()
     def test_npm_version_latest(self):
-        url = 'npm://registry.npmjs.org;package=@savoirfairelinux/node-server-example;version=latest'
-        fetcher = bb.fetch.Fetch([url], self.d)
+        url = ['npm://registry.npmjs.org;package=@savoirfairelinux/node-server-example;version=latest']
```

Ruff (0.8.2)

Local variable `url` is assigned to but never used

Remove assignment to unused variable `url`

(F841)